pages/participantinfo.tsx

```diff
@@ -64,337 +75,918 @@ export default function ParticipantInfo({
+                        <Checkbox
+                            key="counselling"
+                            isChecked={counseling}
+                            onChange={(e) => {
+                                setCounseling(e.target.checked);
+                                if (e.target.checked) {
+                                    participantTherapy.push(therapy.COUNSELING);
+                                    setParticipantTherapy(participantTherapy);
+                                } else {
+                                    const newParticipantTherapy =
+                                        participantTherapy.filter(
+                                            (th) => th != therapy.COUNSELING,
+                                        );
+                                    setParticipantTherapy(
+                                        newParticipantTherapy,
+                                    );
+                                }
+                            }}
+                        >
+                            Psychotherapy/Counseling
+                        </Checkbox>
+                        <Checkbox
+                            key="art"
+                            isChecked={artTherapy}
+                            onChange={(e) => {
+                                setArtTherapy(e.target.checked);
+                                if (e.target.checked) {
+                                    participantTherapy.push(therapy.ART);
+                                    setParticipantTherapy(participantTherapy);
+                                } else {
+                                    const newParticipantTherapy =
+                                        participantTherapy.filter(
+                                            (th) => th != therapy.ART,
+                                        );
+                                    setParticipantTherapy(
+                                        newParticipantTherapy,
+                                    );
+                                }
+                            }}
+                        >
+                            Music or Art Therapy
+                        </Checkbox>
+                        <Checkbox
+                            key="otherTherapies"
+                            isChecked={hasOtherTherapy}
+                            onChange={(e) =>
+                                setHasOtherTherapy(e.target.checked)
+                            }
+                        >
+                            Other
+                        </Checkbox>
+                        {otherTherapyDetails}
+                    </Stack>
+                </FormControl>
+                <FormControl id="parent-guardian-expectations">
+                    <FormLabel>Parent/Guardian Expectations</FormLabel>
+                    <Textarea
+                        placeholder="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Facilisi mauris enim, egestas."
```

Comment:
Is this the placeholder in the Figma? Maybe you can ask @stacykwok @christinaruu what the text here should be.